homeassistant/components/google_assistant_sdk/__init__.py

```diff
@@ -57,7 +71,11 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
         raise ConfigEntryNotReady from err
     except aiohttp.ClientError as err:
         raise ConfigEntryNotReady from err
-    hass.data.setdefault(DOMAIN, {})[entry.entry_id] = session
+    hass.data[DOMAIN][entry.entry_id][DATA_SESSION] = session
+
+    audio_view = GoogleAssistantSDKAudioView(hass)
+    hass.http.register_view(audio_view)
+    hass.data[DOMAIN][entry.entry_id][DATA_AUDIO_VIEW] = audio_view
```

Comment:
A more commonly seen pattern is to use something like an audio storage manager, rather than passing around references to the View directly. Other code can stick stuff in the manger and the view can interact with that.

(You could also consider having that intermediate manager thing be single object that holds the session, though not sure)

Reply:
Done.